test/e2e_node/util.go

```diff
@@ -387,7 +387,11 @@ func findRunningKubletServiceName() string {
 
 func restartKubelet() {
 	kubeletServiceName := findRunningKubletServiceName()
-	stdout, err := exec.Command("sudo", "systemctl", "restart", kubeletServiceName).CombinedOutput()
+	// reset the kubelet service start-limit-hit
+	stdout, err := exec.Command("sudo", "systemctl", "reset-failed", kubeletServiceName).CombinedOutput()
+	framework.ExpectNoError(err, "Failed to restart kubelet with systemctl: %v, %v", err, stdout)
```

Comment:
Maybe update the comment so it isn't equal to the one below, so debugging becomes easier?

Otherwise this i think er can get this merged! Thanks for catching!

Reply:
nice catch, done